src/main/java/uk/gov/companieshouse/overseasentitiesapi/mapper/TrustDataMapper.java

```diff
@@ -0,0 +1,12 @@
+package uk.gov.companieshouse.overseasentitiesapi.mapper;
+
+import org.springframework.stereotype.Component;
+import uk.gov.companieshouse.overseasentitiesapi.exception.ServiceException;
+import uk.gov.companieshouse.overseasentitiesapi.model.dao.trust.TrustDataDao;
+import uk.gov.companieshouse.overseasentitiesapi.model.dto.trust.TrustDataDto;
+
+@Component
+public interface TrustDataMapper {
```

Comment:
I'd get rid of the interface and just make the TrustDataMapper the implementation as looks like we will only ever have 1 implementation so interface not needed.